changelog-2024.07.08-3.patch

```diff
@@ -0,0 +1,30 @@
+diff --git a/packaging/app.organicmaps.desktop.metainfo.xml b/packaging/app.organicmaps.desktop.metainfo.xml
+index b7d2f18040..e058aae357 100644
+--- a/packaging/app.organicmaps.desktop.metainfo.xml
++++ b/packaging/app.organicmaps.desktop.metainfo.xml
+@@ -111,6 +111,25 @@
+   </screenshots>
+ 
+   <releases>
++   <release version="2024.07.08-3" date="2024-07-08">
++     <description>
++       <p>Highlights:</p>
++         <ul>
++           <li>Better address search in the USA based on US Census TIGER data</li>
++           <li>Fixed rendering of texts in Indic, Arabic, Thai, Lao, Khmer, Myanmar, Tibetan, Hangul, Hebrew scripts</li>
++           <li>OSM data as of July 2, 2024</li>
++           <li>Display major roads earlier on the world overview map</li>
```

Comment:
- A selected place now shows more types/categories/amenities

Reply:
- Display good quality fine gravel pedestrian paths as white dashed
- Support more road/path surface types (bricks, stone, grass paver, soil)
- Mountain search also shows volcanoes
- Better search results for faraway places
- Search now finds places with a number in their names
- Find parcel lockers using their ref numbers

Reply:
Done